Cabal/tests/ParserTests/regressions/mixin-2.cabal

```diff
@@ -0,0 +1,13 @@
+cabal-version: 2.4
+name: mixin
+version: 0
+
+-- mixin field:
+-- in 2.2 we got leading/trailing commas
+-- in 2.2 we got lax space parsing
+executable str-example
+  main-is:             Main.hs
+  build-depends:       base, str-string, str-bytestring
+  mixins:              str-string     ( Str as Str.String     ),
+                       str-bytestring ( Str as Str.ByteString ),
```

Comment:
Can we test the `hiding` variant, as well? That's also changed.

Reply:
Feel free to add.

Reply:
In fact, yes, we should add the test. But I don't have the bandwidth to do it.